### PR TITLE
feat: add seo matrix

### PR DESCRIPTION
**Pull request by zhouzilong2020**
1. cloud provider
2. cloud service
3. database 
4. charge type
5. machine type

### CODE REVIEW

File: frontend/seo/matrix.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { useDBInstanceStore } from "../src/stores/dbInstance";
+import { CloudProvider } from "../src/types";
+
+export const cloudProviderList: string[] = ["AWS", "GCP"];
```
Comment (tianzhou) [inline frontend/seo/matrix.ts:4]: We may need to define a structure with name and id, where id is used as the URL.

File: frontend/seo/matrix.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+) => {
+  const dbInstanceStore = useDBInstanceStore();
+  dbInstanceStore.loadDBInstanceList();
+  const dbInstance = dbInstanceStore.dbInstanceList;
+
+  const _machineTypeList: string[] = [];
+  dbInstance.forEach((instance) => {
+    if (instance.cloudProvider === cloudProvider) {
+      if (cloudProvider === "GCP") {
+        // e.g. db-N1Standard-96-360 -> N1Standard
```
Comment (tianzhou) [inline frontend/seo/matrix.ts:33]: use lowercase and either keep db prefix or remove it for all cloud providers.

Comment (zhouzilong2020) [reply frontend/seo/matrix.ts:33]: Should we apply lower case for all the params in URL?

Comment (tianzhou) [reply frontend/seo/matrix.ts:33]: SG

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**